cli/constants.py

```diff
@@ -3,6 +3,8 @@
 """
 ADDRESS_REGEXP = '[0-9a-f]{70}'
 
+BATCH_ID_REGEXP = '[0-9a-f]{128}'
```

Comment:
Remove blank line between the same entities.

Reply:
Done.